examples/pwm.rs

```diff
@@ -23,7 +23,7 @@ fn main() -> ! {
     // Configure our clocks
     let mut flash = dp.FLASH.constrain();
     let mut rcc = dp.RCC.constrain();
-    let clocks = rcc.cfgr.freeze(&mut flash.acr);
+    let clocks = rcc.cfgr.sysclk(16.mhz()).freeze(&mut flash.acr);
```

Comment:
I like the change, as being more explicit about the clock configuration for this example makes sense. 
It wouldn't be needed, if the bug I mentioned above would not exist, but this does not hurt.